stable/minio/templates/minio_configmap.yaml

```diff
@@ -0,0 +1,164 @@
+{{- if eq .Values.mode "distributed" }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}-cm
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+{{ $nodeCount := .Values.replicas | int64 }}
+data:
+  start.sh: |
+     #!/bin/sh
+     {{- if .Values.configPath }}
+     minio -C {{ .Values.configPath }} server \
+     {{- else }}
+     minio server \
+     {{- end }}
+     {{- if eq $nodeCount 4 }}
```

Comment:
Thanks @prydonius ! I was looking to add a loop but wasn't able to get it working. I will make these changes.